packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/feature/redirect/redirect.controller.js

```diff
@@ -135,20 +139,22 @@ export default class TelecomTelephonyAliasConfigurationRedirectCtrl {
   }
 
   canDestinationBeUsedForPresentation() {
-    const trunkOffer = 'trunk';
-    const faxFeatureType = 'fax';
-    const regExp = new RegExp(trunkOffer);
+    const regExp = new RegExp(FEATURE_TYPES.TRUNK);
     if (this.newDestination) {
       return (
         !regExp.test(
-          get(this.newDestination, 'getPublicOffer.name', trunkOffer),
-        ) && this.newDestination.featureType !== faxFeatureType
+          get(this.newDestination, 'getPublicOffer.name', FEATURE_TYPES.TRUNK),
```

Comment:
**question:** If `getPublicOffer.name` is not present in `this.newDestination`, default value of `FEATURE_TYPES.TRUNK` will be returned and it will pass the regex test. Is it intended? If not, make the following change,
```suggestion
          get(this.newDestination, 'getPublicOffer.name'),
```

Reply:
I updated the code, thank you for your comment